avocado-plugins-vt.spec

```diff
@@ -7,15 +7,15 @@
 Summary: Avocado Virt Test Plugin
 Name: avocado-plugins-vt
 Version: 51.0
-Release: 0%{?dist}
+Release: 1%{?dist}
 License: GPLv2
 Group: Development/Tools
 URL: http://avocado-framework.readthedocs.org/
 Source0: https://github.com/avocado-framework/%{modulename}/archive/%{commit}/%{modulename}-%{version}-%{shortcommit}.tar.gz
 BuildRequires: python2-devel, python-setuptools
 BuildArch: noarch
 Requires: avocado >= 36.4
-Requires: python, autotest-framework, xz, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, aexpect, git, python-netaddr, python-netifaces, python-simplejson
+Requires: python, autotest-framework, xz, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, python-aexpect, git, python-netaddr, python-netifaces, python-simplejson
```

Comment:
@ldoktor the current SPEC expects to be built in a mock environment where avocado's official repo is enabled.  From the `Makefile`:

```
# NOTE: to build Avocado-vt RPM packages extra deps not present out of the box
# are necessary. These packages are currently hosted at:
#
# - https://repos-avocadoproject.rhcloud.com/static/avocado-fedora.repo
# - https://repos-avocadoproject.rhcloud.com/static/avocado-el.repo
#
# Since the RPM build steps are based on mock, edit your chroot config
# file (/etc/mock/<your-config>.cnf) and add the COPR repo configuration there.
#
```

There's a slight mistake there about COPR, but you get the point.

So, only the `python-avocado` packages from that repo have the `Provides: avocado` line.  We can indeed improve on this later, though.